sentry/src/main/java/io/sentry/HubAdapter.java

```diff
@@ -215,7 +215,6 @@ public void flush(long timeoutMillis) {
 
   @Override
   public @NotNull ISentryLifecycleToken makeCurrent() {
-    // TODO [HSM] this wouldn't do anything since it replaced the current with the same Scopes
```

Comment:
Yes, at least for cross platform SDKs until they support different scope types separately themselves.